src/main/java/ca/uhn/fhir/jpa/starter/Application.java

```diff
@@ -52,7 +55,7 @@ public ServletRegistrationBean hapiServletRegistration() {
     JpaRestfulServer jpaRestfulServer = new JpaRestfulServer();
     beanFactory.autowireBean(jpaRestfulServer);
     servletRegistrationBean.setServlet(jpaRestfulServer);
-    servletRegistrationBean.addUrlMappings("/fhir/*");
+    servletRegistrationBean.addUrlMappings("/fhir/rest/*");
```

Comment:
I am not sure why this change was made. Could you help me out?

Reply:
@ekigamba this change was made to make the test page overlay work. 
As context for web UI and API both were same starting with `/fhir` therefore we have added `/fhir/rest `to distinguish the context for the APIs. With this approach, we have disabled the web security on /fhir. 

But test page overlay still does not support Authentication and is broken when we try to hit API from there. We may think to remove it as it was before and fix the test page overlay in some other issue. 

Let me re-test it after removing the `/rest` from the context.

cc: @maimoonak  @dubdabasoduba

Reply:
Cool

Reply:
Updated back to the original one. @ekigamba 
cc : @maimoonak